DSchau/ringme.js
# Grooming/eslint

## Helps on #32 by @DSchau 

### Changes proposed in this pull request:

* Lets ignore node modules installation folder
* Make sure to get a _known_ state by adding a _lock file_ of installed node modules and there versions
* Add a contributing guideline to explicit the fact that the code needs to pass ESLint

### Status


### How to verify this change

* Check that ESLint still passes with those changes
* Check that npm modules installation folder is ignore by git
* Verify that the ESLint contribution guideline appears properly in [contributing file](https://github.com/ssirois/ringme.js/blob/grooming/eslint/CONTRIBUTING.md)
